client_plugin/utils/config/config.go

```diff
@@ -58,6 +60,7 @@ type Config struct {
 	LogPath        string `json:",omitempty"`
 	MaxLogSizeMb   int    `json:",omitempty"`
 	MaxLogAgeDays  int    `json:",omitempty"`
+	MaxLogFiles    int    `json:",omitempty"`
```

Comment:
1 x 10MB or 10  x 1MB doesn't matter, the log just rotates when the max log size has been achieved. Configuring the max log files seems like something the user doesn't have to do. Why would the user care how the MaxLogSizeMb is sliced into how many files?

The fix for this issue is to cap the MaxLogSizeMb to something reasonable (vFile had 10 x 100MB, which is too large). Say 100MB cap on the max log size is sufficient vs. adding a new config.

Reply:
> I'd suggest not adding a new config and instead cap the max log size instead to a reasonable maximum.

```
	// MaxBackups is the maximum number of old log files to retain.  The default
	// is to retain all old log files (though MaxAge may still cause them to get
	// deleted.)
	MaxBackups int `json:"maxbackups" yaml:"maxbackups"`
```

With the above reference, I agree it can be controlled through `MaxAge` but it doesn't favor always. By taking example of https://github.com/vmware/docker-volume-vsphere/issues/1959#issue-269565208, `MaxAge` doesn't help always, let's say vFile/vDVS plugin spits out log heavily, disk will be filled up quickly whereas it can be avoided through `MaxBackups` config option.

Reply:
Not MaxAge, MaxLogSize is what I'm referring to, fix the max log size and the number of files doesn't matter.

Reply:
@govint As discussed offline, `MaxLogSize ` doesn't help alone when `MaxAge ` is in place. 

some more reference:

MaxSize=10MB
MaxAge=28 days
MaxBackup=3

if plugin behaves crazy .. there will be max 3 files 30MB at any given time .. otherwise MaxAge expires one as its value.

Please let me know if there is any more comment.